app/components/hero/Hero.module.css

```diff
@@ -1,19 +1,179 @@
 .container {
   position: relative;
-  z-index: 2;
+  z-index: 12;
   width: 100%;
-  max-width: 1920px;
-  height: 100vh;
+  min-width: 320px;
+  max-width: 1728px;
+  height: 100svh;
+  min-height: 700px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.bottomContainer {
+  position: absolute;
+  grid-area: 7;
+  top: 10px;
+  width: 70vw;
+  max-width: 510px;
+  min-height: calc(100svh - 200px);
+  max-height: 980px;
+  display: flex;
+  flex-direction: column;
+  left: 50%;
+  transform: translateX(-50%);
+  justify-content: space-between;
+}
+
+.bottomContainer * {
```

Comment:
`*` je z hlediska udrzitelnosti bad practice, kor u specifickejch vlastnosti, jako jsou marginy. Pokud vim, tak to ovlivnuje jenom ty buttony, tak bych to potrebny zarovnani vyresil tam.

Reply:
oddelal jsem